mud_manager.c

```diff
@@ -219,8 +272,24 @@ static int read_mudmgr_config (char* filename)
         acl_list_type = INGRESS_ONLY_ACL;
     }
 
-    //MUDC_LOG_INFO("MUDCTRL CA Cert <%s> MUDCTRL Cert <%s> MUDCTRL Key <%s>", mudmgr_CAcert, mudmgr_cert, mudmgr_key);
     {   // moved if (!config_json) test earlier; skip moving below lines for now...
+      if ((vlan_json = cJSON_GetObjectItem(config_json, "VLANs")) != NULL) {
+	for (i=0;i< cJSON_GetArraySize(vlan_json);i++) {
+	  tmp_json=cJSON_GetArrayItem(vlan_json,i); /* fill out the pool */
+	  vlan_list[i].vlan=GETINT_JSONOBJ(tmp_json,"VLAN_ID");
+	  vlan_list[i].v4_nw=GETSTR_JSONOBJ(tmp_json,"v4addrmask");
+	  vlan_list[i].v6_nw=GETSTR_JSONOBJ(tmp_json,"v6addrmask");
+	  /* Sanity check */
+	  if (vlan_list[i].vlan == 0 || // no vlan entry should be 0
```

Comment:
It would be good to have individual MUDC_LOG_ERR() messages for each of the errors so that the user knows how to correct the problem.

Reply:
Ok, let me open a separate issue on that.

Reply:
Actually I committed some code for this.

Reply:
but then forgot to push it.